docs/content/flags.md

```diff
@@ -41,13 +41,13 @@ These flags are available for every command.
       --dump-bodies                          Dump HTTP headers and bodies - may contain sensitive info
       --dump-headers                         Dump HTTP headers - may contain sensitive info
       --exclude stringArray                  Exclude files matching pattern
-      --exclude-from stringArray             Read exclude patterns from file
+      --exclude-from stringArray             Read exclude patterns from file (use - to read from stdin).
```

Comment:
This file is auto generated. You'll need to change the original here

https://github.com/rclone/rclone/blob/37d5e75a569e063aa181f440e08ccc9e0cb1567f/fs/filter/filterflags/filterflags.go#L25